lib/tasks/integration.rake

```diff
@@ -0,0 +1,68 @@
+namespace :integration do
+  desc "Seed the database and configure assets for integration tests"
+  task :seed => ['seed:all']
+
+  namespace :seed do
+    task :all    => [:db, :assets]
+    task :db     => [:env, "test:vmdb:setup", "evm:foreman:seed"]
+    task :assets => [:env, "test:initialize", :compile_assets]
```

Comment:
Do these need `desc`?  I know we talked about the need to sometimes just seed the db and not the assets.

Reply:
Yeah, I wasn't sure about that.  It was one of those things that I figured with what you were doing, that was the exception of to the "happy path", so I didn't want them to show up in `rake -T` and are intended just to be support tasks.

But if you think otherwise, I can change.